extra-libs/uthash/spec

```diff
@@ -0,0 +1,3 @@
+VER=2.1.0
+SRCTBL="https://github.com/troydhanson/uthash/archive/v$VER.tar.gz"
```

Comment:
Please switch to our new `spec` format, as specified in our [ACBS Documentation](https://wiki.aosc.io/developer/packaging/acbs/spec-format/). An example format for your reference:

```
VER=2.1.0
SRCS="https://github.com/troydhanson/uthash/archive/v$VER.tar.gz"
CHKSUMS="sha256::152ccd8e64d0f495377232e3964d06c7ec8bb8c3fbd3217f8a5702614f9a669e"
```